src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -953,6 +971,33 @@ public static Completable onAssembly(Completable source) {
         }
     }
 
+    /**
+     * Wraps the call to the callable in try-catch and propagates thrown
+     * checked exceptions as RuntimeException.
+     * @param <T> the input type
+     * @param <T> the output type
+     * @param t the callable, not null (not verified)
+     * @return the result of the callable call
+     */
+    static <T> T call(Callable<T> t) {
```

Comment:
Not sure what you mean. I've tried to be consistent with the abstraction of the similar `apply` method. Do you think that should change?